app/controllers/supported_demands_controller.rb

```diff
@@ -31,6 +30,6 @@ def update_params
   end
 
   def find_demand
-    user_municipality.demands.find(params[:id])
+    Demand.find(params[:id])
```

Comment:
I think if we go the materialized view route then we can use that to filter the scope we'll use to find the demand. I think this is 👍 for now.